.github/workflows/eks-beta-add-on-integ-test.yml

```diff
@@ -76,6 +76,17 @@ jobs:
           role-to-assume: ${{ env.TERRAFORM_AWS_ASSUME_ROLE }}
           aws-region: us-west-2
 
+      - name: Confirm EKS Version Support
```

Comment:
Can we add this to the prod eks add-on integ test too?  https://github.com/aws/amazon-cloudwatch-agent-operator/blob/main/.github/workflows/eks-add-on-integ-test.yml

Reply:
Done